File: critreel/critreel_codigo/critreel_ionic/projeto/src/app/model/filme.ts
export class Filme {
    idFilme: number;
    titulo: string;
    sinopse: string;
    linkTrailer: string;
    fotosCenas: string; //RETIRADO DA IMPLEMENTAÇÃO
    anoLancamento: string;
    idUsuario: number;
    linkAssistir: string;

    constructor() {
        this.idFilme = 0;
        this.titulo = '';
        this.sinopse = '';
        this.linkTrailer = '';
        this.fotosCenas = ''; //RETIRADO DA IMPLEMENTAÇÃO
        this.anoLancamento = '';
        this.idUsuario = 0;
        this.linkAssistir = '';
    }
}

File: critreel/critreel_codigo/critreel_api/critreelFinal/critreel/src/main/java/br/cefetmg/critreel/model/Filme.java
package br.cefetmg.critreel.model;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Filme {
    private Long idFilme;
    private String titulo;
    private String sinopse;
    private String linkTrailer;
    private String anoLancamento;
    private Long idUsuario;
    private String linkAssistir;
}
